Pull Request for the branch (home-page)

HamzaAnwar1998 opened the pull request:
This branch also contains the about page code.

- CSS flexbox is used to create this UI
- In the Featured Speakers section, the speakers are rendering dynamically using an array of objects
- Live demo link will work after I merge this branch to the main

Link to the loom video
https://www.loom.com/share/de792cf575784fef9b5ec3328c8ddb55

HamzaAnwar1998 committed:
create home page | create about page

HamzaAnwar1998 committed:
fix stylelint errors

HamzaAnwar1998 committed:
add and fix hint errors

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,151 @@
+
+        <!-- hero -->
+        <section class="hero-box">
+            <div class="small-headline">"Hello! Sharing World"</div>
```
- [ ] Please add the background image before this text in your desktop version, Currently, I can only see the background-image on the mobile view👍🏿 

- [ ] Please feel free to personalize the content of your website, you can use a P tag instead of a div also add your own text 😉 

Behance Design          |  Desktop design     
:-------------------------:|:-------------------------:
![hello](https://user-images.githubusercontent.com/69638013/157675947-2ab85d98-37c7-4328-99e2-3ace7cf1057b.png)  |  ![hello 2](https://user-images.githubusercontent.com/69638013/157675892-5ec91c5c-ae73-4329-a6a4-80d11b3d3351.png)

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,151 @@
+        <!-- end of partners -->
+
+        <!-- footer -->
+        <footer class="footer">
+            <a class="logo-link" href="/index.html">
+                <i class="fa-solid fa-code fa-xl"></i>
+                <span class="footer-logo-text">js Solutions</span>
+            </a>
+            <p class="rights-reserved">2022 JS SOLUTIONS. ALL RIGHTS RESERVED</p>
+        </footer>
```
- [ ] Kindly align the logo to the left and align the text to the right in the desktop version.

prolajumokeoni commented on about.html:
```diff
@@ -0,0 +1,136 @@
+
+        <!-- hero -->
+        <section class="hero-box about">
+            <h1 class="headline about">GET YOUR JS SOLUTIONS <br>2022</h1>
```
- [ ]  Kindly change this h1 tag to h2 tag, It makes your website easier to read and more semantic by screen readers, You should have just one h1 tag in your project 👍🏿

HamzaAnwar1998 replied:
but this is the only h1 tag I used on the entire page, I would really appreciate it if you elaborate your point

prolajumokeoni replied:
- You already have one in the index.html which is the main page

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,151 @@
+        <section class="hero-box">
+            <div class="small-headline">"Hello! Sharing World"</div>
+            <h1 class="headline">GET YOUR JS SOLUTIONS <br>2022</h1>
+            <p class="para">JS SOLUTIONS is a YouTube channel and blogging website where you can
```
- [ ] Please add the border outline to make your design more similar to the original 👍🏿

prolajumokeoni commented on styles.css:
```diff
@@ -0,0 +1,779 @@
+.hero-box .headline{
+    font-size: 72px;
+    font-family: 'League Gothic', sans-serif;
+    color: #b70001;
```
- [ ] Kindly use the right font color for your design 👍🏿

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,151 @@
+        <!-- end of hero -->
+
+        <!-- main program -->
+        <section class="main-program">
+            <h3 class="heading">Main Program</h3>
+            <div class="underline"></div>
+            <div class="cards-box">
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-chalkboard-user fa-2xl"></i></span>
+                    <h5 class="card-heading">Lecture</h5>
+                    <p class="card-text">Listen to the speakers about how you can master web development</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-blog fa-2xl"></i></span>
+                    <h5 class="card-heading">Tech Blogs</h5>
+                    <p class="card-text">Read blogs on coding tips and tricks to enhance your knowledge</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-video fa-2xl"></i></span>
+                    <h5 class="card-heading">Video Courses</h5>
+                    <p class="card-text">Watch high quality tutorials to master the concepts of programming</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-message fa-2xl"></i></span>
+                    <h5 class="card-heading">Forum</h5>
+                    <p class="card-text">Have the time to share your thoughts and opinions with experts for each topic</p>
+                </div>
+                <div class="card">
+                    <span class="card-icon"><i class="fa-solid fa-circle-question fa-2xl"></i></span>
+                    <h5 class="card-heading">Ask Questions</h5>
+                    <p class="card-text">Have a question? Get in touch directly with our experts instructors</p>
+                </div>
+            </div>
+            <a class="see-program" href="#">SEE THE WHOLE PROGRAM</a>
+            <a class="join-community" href="#">JOIN THE COMMUNITY</a>
+        </section>
```
- [ ] Kindly add a background image to this section please check the Behance design

prolajumokeoni commented on styles.css:
```diff
@@ -0,0 +1,779 @@
+
+.wrapper{
+    width: 100%;
+    max-width: 1440px;
```
- [ ] Adding max-width to your page creates a left and right padding please remove or update the width of your page

HamzaAnwar1998 replied:
I have changed it but I added max-width to keep the size of my website perfect, this is the same thing that a bootstrap container class does.

prolajumokeoni replied:
- [ ] Yes but if u look closely  above  width 1440px there is spacing check screenshot below, this will also help you look out for other screen sizes above 1440px
![image](https://user-images.githubusercontent.com/69638013/157705376-ea74b86e-b443-470f-b262-357a4b3c367c.png)

prolajumokeoni reviewed (changes requested):
Hi @HamzaAnwar1998 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design and presentation


## Required Changes ♻️
- [ ] Kindly use the right fonts originally used: Cocogoose and Lato for all fonts on your project, please add this font for all headers  

- [ ] Please fix and update the live demo link currently what I see is the readme file when I click on the live demo link 

- [ ] I noticed you added media queries and the media queries are targeted at the mobile view, not the desktop view. It would be better to always start your design with the mobile-first approach. Max width will only make changes below the specified pixels instead use **min-width**  after building the mobile and start adding media queries for the desktop size, I advise grouping all media queries in one block instead of separating it, It makes your media queries easier to manage 👍🏿 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Commit Early And Often because it helps you commit only related changes.  It helps you and  people quickly understand changes made in your code without having to read code



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

prolajumokeoni commented on index.html:
```diff
@@ -0,0 +1,151 @@
+        <section class="featured-speakers-box">
+            <h3 class="box-heading">Featured Speakers</h3>
+            <div class="heading-underline"></div>
+            <div class="speakers-box" id="speakers-box">
```
- [ ] Please add the first image that appears before the speaker image, also the second speaker is missing the staus text please add it 👍🏿

HamzaAnwar1998 replied:
the second speaker status text is also missing from Behance

HamzaAnwar1998 committed:
resolve all changes except mobile first approach

HamzaAnwar1998 committed:
changecss to mobile first approach

HamzaAnwar1998 committed:
social links color changed

HamzaAnwar1998 committed:
remove stylelint errors

HamzaAnwar1998 commented:
- h1 tag is changed to h2 tag in about.html
- removed max-width from the wrapper
- social icons at the top bar have been changed to #D3D3D3
- the background image is added in the main program sections
- changed CSS to a mobile-first approach
- media queries now have a single block
- added the checked image behind the speaker

ReshyResh reviewed (approved):
# Hi @HamzaAnwar1998 , 

### Good job addressing the changes requested by the previous reviewer.
#### Your project is complete!  Although I recommend you take a look at the optional changes I listed below, you can merge your project!

Congratulations! 🎉 

#### [OPTIONAL]

- [ ] The mobile menu in the about page currently doesn't have a menu element that lets the user return to the homepage, it still shows the same options as the main page, I suggest you add a link to 'Home'.
- [ ] It seems like you're getting a console error on the about page, saying speakerbox is undefined. This is happening most likely because you're trying to populate the speakers section on the about page as well, but it should be called only on the main page. For a more professional looking website, I suggest you fix this issue.

    ![](https://i.imgur.com/UTwlOx5.png)


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

HamzaAnwar1998 committed:
add home link | create navjs to remove the console error in aboutjs
